system/ee/ExpressionEngine/Controller/Utilities/MemberImport.php

```diff
@@ -667,6 +667,13 @@ public function doImport()
                 $dupe = $member_obj->getValues();
                 unset($dupe['member_id']);
                 ee('Model')->make('Member', $dupe)->save();
+            } elseif (isset($data['member_id'])) {
```

Comment:
@intoeetive With this PR - when there is a member ID set, it will use that member ID to update the member that had the ID with new data?

I don't like that solution, because (for example) you could end up importing members from one site to another and have it overwrite all existing members when instead the user thinks they are just doing an import.s

I think if we make this change, at the very least we need to warn the user, but I might prefer a solution that ignores the member ID and imports the user anyways since that is the function of this utility.

Reply:
This is how it is expected to work. There is already a banner with working about that on the page